src/main/java/com/jcabi/github/StatusState.java

```diff
@@ -0,0 +1,5 @@
+package com.jcabi.github;
+
+public enum StatusState {
```

Comment:
@zygm0nt statuses in ```Issue``` are already static final, because they are defined in the interface.
I agree with you about enum. It's useful. Let's rename it to ```State``` and put inside ```Status``` interface